desktop/core/src/desktop/middleware.py

```diff
@@ -823,7 +823,7 @@ def process_response(self, request, response):
       if any(regexp.match(location) for regexp in redirection_patterns):
         return response
 
-      if is_safe_url(location, request.get_host()):
+      if is_safe_url(location, allowed_hosts=request.get_host()):
```

Comment:
`allowed_hosts` is a set https://docs.djangoproject.com/en/1.11/_modules/django/utils/http/ ?

i.e. it expects multiple ones and we can't blindly swap the names